### ISSUE TITLE
Rival 600 Issues

### ISSUE THREAD

**Issue by ch4n3l** [flozz/rivalcfg#64]
Note: I've read that there are issues with the Rival 600.

When resetting the mouse, I receive this error:
```
TypeError: rgbuniversal_handler() takes exactly 5 arguments (2 given)
```

**Comment by flozz** [comment]
Hello,

I have no Rival 600 mouse to test, but maybe @ergor can help with this? :)

**Comment by ergor** [comment]
I've submitted a PR with a quick fix

**Comment by flozz** [comment]
The fix is merged, I will probably release a new version this weekend with this fix :)

**Comment by flozz** [comment]
This fix is now released in **rivalcfg 3.1.0**, thanks for your help :)
